src/scripts.js

```diff
@@ -126,7 +127,13 @@ function searchRecipes() {
 }
 
 function toggleSaveRecipe() {
-  currentUser.toggleSaveRecipe(modalRecipe);
+  if(!modalRecipe.saved){
+    apiObject.apiRequest("usersRecipes","POST", currentUser, modalRecipe);
+  } else {
+    apiObject.apiRequest("usersRecipes","DELETE", currentUser, modalRecipe);
+  }
```

Comment:
I think we should move this conditional into the user.toggleSaveRecipe method and then just call that method here.

Reply:
After discussing it, I think this is the best place for it and we can just get rid of the user.toggleSaveRecipe method